### ISSUE TITLE
Package installation fails for Python 3.12 due to missing 'distutils' module

### ISSUE THREAD

**Issue by shayan09** [ahnazary/stockdex#10]
Error:

```
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1310, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1331, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 935, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/tmp/pip-build-env-r4268o3h/overlay/lib/python3.12/site-packages/setuptools/__init__.py", line 10, in <module>
    import distutils.core
ModuleNotFoundError: No module named 'distutils'
```

**Comment by jensvog** [comment]
I get the same error with Python 3.12. `distutils` is no longer shipped with Python 3.12. The dependency comes from `numpy`. I guess with a newer version of `numpy` it should work.

Maybe I can check and provide a Pull Request in the next days or the maintainer might also take a look directly.

**Comment by ahnazary** [comment]
This has been fixed with the new `stockdex==0.6.1` release. will close the issue. thanks @jensvog @shayan09
